client/planning-extension/src/assignments-overview/index.tsx

```diff
@@ -0,0 +1,245 @@
+import {groupBy} from 'lodash';
+import * as React from 'react';
+import {
+    IDesk,
+    IRestApiResponse,
+    IUser,
+    IVocabulary,
+    IVocabularyItem,
+} from 'superdesk-api';
+import {IAssignmentItem} from '../../../interfaces';
+import {superdesk} from '../superdesk';
+import {Badge} from 'superdesk-ui-framework';
+import {AssignmentsOverviewListItem} from './assignments-overview-list-item';
+
+const {addWebsocketMessageListener} = superdesk;
+
+const DropdownTree = superdesk.components.getDropdownTree<IAssignmentItem>();
+const {queryRawJson, findOne} = superdesk.dataApi;
+const {GroupLabel, IconBig, TopMenuDropdownButton} = superdesk.components;
+
+interface IState {
+    loading: false;
```

Comment:
What is the benefit of using the values `false` or `true` here instead of the typical `boolean`?